data/seeds/virtualChild.js

```diff
@@ -0,0 +1,13 @@
+
+exports.seed = function(knex) {
+  // Deletes ALL existing entries
+  return knex('VirtualChild').del()
+    .then(function () {
+      // Inserts seed entries
+      return knex('VirtualChild').insert([
+        {id: 1, email: '', first_name:"Mike",last_name:"McVirtual"},
+        {id: 2, email: '', first_name:"Sara",last_name:"McVirtual"},
+        {id: 3, email: '', first_name:"Jamie",last_name:"McVirtual"}
```

Comment:
Since we're going to be inserting this seed into the Childrens table, we're going to need to fill in all required fields.
You can take a quick look at these fields here: https://app.dbdesigner.net/designer/schema/0-scribble-stadium
Also, we don't need to have this "id" bit here, because it will automatically increment.